contracts/libraries/RevertReasonForwarder.sol

```diff
@@ -6,7 +6,8 @@ pragma abicoder v1;
 library RevertReasonForwarder {
     function reRevert() internal pure {
         // bubble up revert reason from latest external call
-        assembly {  // solhint-disable-line no-inline-assembly
+        /// @solidity memory-safe-assembly
```

Comment:
This is unsafe example from solidity docs https://docs.soliditylang.org/en/v0.8.15/assembly.html?highlight=memory%20safe%20assembly#memory-safety